lib/phoenix_component/declarative.ex

```diff
@@ -817,21 +817,20 @@ defmodule Phoenix.Component.Declarative do
   defp build_attrs_docs(attrs) do
     [
       "## Attributes\n",
-      for attr <- attrs, attr.doc != false, attr.type != :global, into: [] do
```

Comment:
We were skipping proper generation for `:global` here...

Reply:
I think it was written in this style so the `:global` attribute always comes last. Should we revert this particular change?

Reply:
That does make sense. I'll revert that part, then and then we can follow up on performance not potentially being enough

Reply:
I skipped global in the first for pass, then did an enum find at the end for the global. This means that only one global actually gets docs generated. I'm not sure if multiple globals should be supported?

It also means potentially 2 passes through the attrs list so not as performant as it technically could be, but trying to do it all in one pass means a reduce or some other less clear alternative.